src/examples/v2/fund-transfer.ts

```diff
@@ -0,0 +1,58 @@
+import { Bool, Experimental, Int64, PrivateKey, PublicKey, sendZkapp, UInt32, UInt64 } from 'o1js';
+
+const {
+  ZkappCommand,
+  ZkappFeePayment,
+  AccountUpdate,
+  AccountUpdateAuthorizationKind,
+  GenericData,
+  AccountId,
+  TokenId,
+} = Experimental.V2;
+
+// Generate a random keypair for the fee payer
+const feePayerKey = PrivateKey.fromBase58('EKEDzWYFqRrF8sKxeD3ibLy2rcxCe4UoREgKxrKNUvNbh1d8dtt2'); // WARNING: This is a hardcoded private key, do not use in production
+const feePayerAddress = feePayerKey.toPublicKey(); // B62qjTDdVdxkkRNRbkpdq6D9RUvX2C7RUaW3voHMAjtmk6WLxuXHfS9
+
+const receiverAddress = PrivateKey.random().toPublicKey();
+
+const amount = 11000000n;
+const fee = 10n * 10n ** 9n;
+
+const accountCreationFee = 1000000000n;
+
+const totalAmount = amount + fee;
+
+// build a manual zkapp command that transfers funds from the fee payer to the receiver address
+const paymentCommand = await new ZkappCommand({
+  feePayment: new ZkappFeePayment({
+    publicKey: feePayerAddress,
+    fee: new UInt64(fee),
+    nonce: new UInt32(4),
+  }),
+  accountUpdates: [
+    new AccountUpdate('GenericState', GenericData, GenericData, {
```

Comment:
feels tough to require a state + events + action layout on _every_ account update. this example shows why: often you'll create AUs where state/events/actions don't matter.

maybe there should be two different constructors? `AccountUpdate` and `ZkappAccountUpdate` or similar? and only the latter requires the layouts?